source/reference/cutover-process.txt

```diff
@@ -114,41 +115,18 @@ Steps
       ``true``. If ``canWrite`` is ``false``, wait until ``progress``
       shows ``canWrite`` is ``true``.
 
-      Request
-      ~~~~~~~
+      .. io-code-block::
+         :copyable: true
 
-      .. literalinclude:: /includes/api/requests/progress.sh
-         :language: shell
-
-      Response
-      ~~~~~~~~
+         .. input::
+            :language: bash
 
-      .. code-block:: json
-         :emphasize-lines: 6
-         :copyable: false
+            curl localhost:27182/api/v1/progress -XGET | jq ".progress.canWrite"
```

Comment:
I wonder if it's better to add `-sS` flag so that `curl` doesn't print the progress bar.

Reply:
Fixed